src/main/java/org/generationcp/middleware/api/brapi/v2/observationunit/ObservationUnitServiceImpl.java

```diff
@@ -317,6 +321,21 @@ public List<String> importObservationUnits(final String crop, final List<Observa
 		return observationUnitDbIds;
 	}
 
+	private void checkIfExistsObsUniIdVariable(final Map<Integer, DmsProject> plotDatasetMap) {
```

Comment:
@cuenyad the name is not clear enough, it is not only checking right? let's find a name